claude/skills/sandbox-validation/SKILL.md

```
@@ -0,0 +1,140 @@
+
+**Usage:**
+```powershell
+# Basic validation
+./scripts/Validate-SandboxEnvironment.ps1 -PersonalAccessToken $pat
+
+# Custom organization
+./scripts/Validate-SandboxEnvironment.ps1 -PersonalAccessToken $pat -Organization "https://myorg.visualstudio.com"
+```
+
+## Examples
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)